README.md

```diff
@@ -59,12 +59,12 @@ def helloWorld():
 
 #### Logging
 
-Flask-Cors uses standard Python logging, using the module name 'Flask-Cors'. You can read more about logging from [Flask's documentation](http://flask.pocoo.org/docs/0.10/errorhandling/).
+Flask-Cors uses standard Python logging, using the logger name '`app.logger_name`.cors'. The app's logger name attribute is usually the same as the name of the app. You can read more about logging from [Flask's documentation](http://flask.pocoo.org/docs/0.10/errorhandling/).
```

Comment:
Agreed on all points. I wish there was a clearer pattern for logging in Flask extensions. 

In this case, we could refactor the __init__ or init_app to take and save a logger instance derived from the passed app. Perhaps that is in fact the best approach, but I have a negative gut reaction to always using bound functions. It tends to complicate things from a composition perspective, in my experience.